resolver/blocking_resolver.go

```diff
@@ -296,6 +296,12 @@ func (r *BlockingResolver) BlockingStatus() api.BlockingStatus {
 	}
 }
 
+func (r *BlockingResolver) GetHealth() api.HealthStatus {
+	return api.HealthStatus{
+		Version: util.Version,
```

Comment:
Is `util.Version` the best value?

Reply:
yes `util.Version" is ok, but the implementation should not be in "blocking_resolver.go", since this operation has nothing to do with blocking. server_endpoints.go would be a better place